tests/unit/test_grpc_interpreter.py

```diff
@@ -0,0 +1,145 @@
+import numpy
+
+from nidaqmx._grpc_interpreter import _assign_numpy_array
+from nidaqmx._stubs import nidaqmx_pb2
+
+
+def test__grpc_array_with_all_samples__update_inuput_array__all_samples_updated():
+    number_of_samples_in_input = 100
+    number_of_samples_read = 100
+    initial_value = -1
+    read_value = 100
+    input_array = numpy.full(number_of_samples_in_input, initial_value, dtype=numpy.float64)
+    output_array = numpy.full(number_of_samples_in_input, read_value, dtype=numpy.float64)
+    response = nidaqmx_pb2.ReadAnalogF64Response(read_array=output_array)
+
+    _assign_numpy_array(input_array, response.read_array, number_of_samples_read)
+
+    assert all(element == read_value for element in input_array)
```

Comment:
I think this is more numpy-esque:
```
assert (read_value == input_array).all()
assert (read_value == input_array[:number_of_samples_read]).all()
```
etc.

== returns an ndarray of bool and all() checks that all of the bools in the array are true. I see you are also using this technique with count_nonzero, which is good.

I would hope that this allows pytest -vvv to print the ndarray of bools when the test fails.